docs/developer-experience/readme.md

```diff
@@ -0,0 +1,231 @@
+# bauta.js and developer experience
+
+## Audience of this document
+
+This document is for anyone that is starting to learn about bauta.js library. For example a developer that has just discovered bauta.js or a techical lead in a team that is wondering whether bauta.js could be helpful to them or cover their needs.
+
+## Reason for this document
+
+The goal of this document is to help you to answer the question: "Do I want to use bauta.js?" and provide you a starting point answering the question "How do i get started with bauta.js?".
+
+We hope that by reading this document, that should not take more than 15 or 20 minutes, you can review quickly what is bauta.js about and let you determine if it could be helpful to you.
+
+This document tries to convey the model used by bauta.js at high level. This is to speed up the process of understanding how bauta.js is used and if it is useful to you.
+### What this document is *not*
+
+- This document does not list bauta.js capabilities or features. 
+- This document is not a tutorial on the library itself, nor it is a hands-on. You can have that information in the documentation starting [here](../../README.md).
+- Since this is not a tutorial, the explanations in this section differ a bit from the common pattern "piece of code" --> "what stuff does each line". 
```

Comment:
```suggestion
- This document does not list bauta.js capabilities or features. 
- This document is not a tutorial on the library itself nor a hands-on. You can have that information in the documentation starting [here](../../README.md).
- Since this is not a tutorial, the explanations in this section differ a bit from the typical pattern "piece of code" --> "what stuff does each line". 
```